src/main/java/space/obminyashka/items_exchange/controller/AdvertisementController.java

```diff
@@ -161,7 +161,10 @@ public AdvertisementModificationDto createAdvertisement(
         final var compressedImages = images.parallelStream()
                 .map(imageService::compress)
                 .toList();
-        return advertisementService.createAdvertisement(dto, owner, compressedImages);
+        byte[] scaledTitleImage = images.stream().findFirst()
+                .map(imageService::scale)
+                .orElse(new byte[0]);
```

Comment:
Maybe should use standard image converted to bytes instead of empty array

Reply:
I think method .orElse(new byte[0]) is unnecessary because a minimum size of the list is 1

Reply:
@ennnbaev if I understood you correctly, we need .orElse, because method scale returns an optional, and I agree with Illya: I will return **compressedImages.get(0)**, or were you talking about something else?